src/sentry/api/endpoints/project_create_sample.py

```diff
@@ -0,0 +1,23 @@
+from __future__ import absolute_import
+
+from rest_framework.response import Response
+
+from sentry.api.bases.project import ProjectEndpoint, ProjectPermission
+from sentry.api.serializers import serialize
+from sentry.utils.samples import create_sample_event
+
+
+class ProjectCreateSampleEndpoint(ProjectEndpoint):
+    permission_classes = (ProjectPermission, )
+
+    def post(self, request, project):
+        if project.platform:
+            event = create_sample_event(
+                project, platform=project.platform, default='javascript', level=0
+            )
+        else:
```

Comment:
In the event the project.platform doesn't have a language that has a sample json, (C# or Elixir for example), I'd like it to default to JavaScript.